manifests/params.pp

```diff
@@ -5,7 +5,7 @@
 class selinux::params {
   $module_build_root = "${facts['puppet_vardir']}/puppet-selinux"
 
-  case $facts['osfamily'] {
+  case $facts['os']['family'] {
     'RedHat': {
       if $facts['operatingsystem'] == 'Amazon' {
```

Comment:
If you want to use the modern facts, then this should probably also be replaced (as well as the one below).